tests/submission/test_upload_signpost.py

```diff
@@ -1,230 +0,0 @@
-"""
```

Comment:
this file should be replaced with new test cases for scenario when ENFORCE_FILE_HASH_UNIQUENESS is False?